app/services/aspace_client.rb

```diff
@@ -72,11 +72,31 @@ def all_published_resource_uris_by(repository_id:)
     published_resource_uris(repository_id:).each.to_a.pluck('uri')
   end
 
+  # Returns an instance of AspaceQuery that response to :each returning
+  # hashes containing an ead id and uri for resources of interest,
+  # e.g. {"ead_id"=>"sc0348.xml", "uri"=>"/repositories/2/resources/5363"}
+  # This query can be slow. It is recommended to pass the result of a faster query to 'uris_to_exclude'.
+  # See https://github.com/sul-dlss/stanford-arclight/issues/551.
+  # This may be able to be removed once on ArchivesSpace v3.5.0+.
+  # @param repository_id [Integer] the repository id in ArchivesSpace
+  # @param updated_after [String] YYYY-MM-DD optionally limit the response to resources updated after a specific date
```

Comment:
Appears to be required.
```suggestion
  # @param updated_after [String] YYYY-MM-DD limits the response to resources updated after a specific date
```